etest/docker/container.py

```diff
@@ -36,7 +54,7 @@ def remove(container, container_name, *args, **kwargs):
     return common.API_CLIENT.remove_container(container_name, *args, **kwargs)
 
 
-def start(container, *args, **kwargs):
+def start(container):
```

Comment:
Sorry I missed this one, but can we add a type hint here too?

Reply:
Sure, I'll add it next time Im tweaking stuff